bit_manipulation/binary_count_trailing_zeros.py

```diff
@@ -0,0 +1,43 @@
+from math import log2
+
+
+def binary_count_trailing_zeros(a: int):
+    """
+    Take in 1 integer, return a number that is
+    the number of trailing zeros in binary representation of that number.
+
+    >>> binary_count_trailing_zeros(25)
+    0
+    >>> binary_count_trailing_zeros(36)
+    2
+    >>> binary_count_trailing_zeros(16)
+    4
+    >>> binary_count_trailing_zeros(58)
+    1
+    >>> binary_count_trailing_zeros(4294967296)
+    32
+    >>> binary_count_trailing_zeros(0)
+    0
+    >>> binary_count_trailing_zeros(-10)
+    Traceback (most recent call last):
+        ...
+    ValueError: the value of input must be positive
+    >>> binary_count_trailing_zeros(0.8)
+    Traceback (most recent call last):
+        ...
+    TypeError: unsupported operand type(s) for &: 'float' and 'float'
+    >>> binary_count_trailing_zeros("0")
+    Traceback (most recent call last):
+        ...
+    TypeError: '<' not supported between instances of 'str' and 'int'
+    """
+    if a < 0:
+        raise ValueError("the value of input must be positive")
```

Comment:
```suggestion
    if a < 0:
        raise ValueError("Input value must be a positive integer")
```
Please use `isinstance` to check whether the input is of type `float` and raise `TypeError` with an appropriate message.

Reply:
sure!